<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Netra.AI</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>20</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuImage_Processing">
    <property name="title">
     <string>Image Processing</string>
    </property>
    <widget class="QMenu" name="menuDetect">
     <property name="title">
      <string>Detect</string>
     </property>
     <addaction name="actionRestricted_Area"/>
    </widget>
    <addaction name="actionFace_Recognition"/>
    <addaction name="menuDetect"/>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QMenu" name="menuFace_Recognition">
     <property name="title">
      <string>Face Recognition</string>
     </property>
     <addaction name="actionConfidencial_Level"/>
     <addaction name="actionCandidates"/>
    </widget>
    <addaction name="menuFace_Recognition"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuImage_Processing"/>
   <addaction name="menuParameters"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfidencial_Level">
   <property name="text">
    <string>Confidencial Level</string>
   </property>
  </action>
  <action name="actionCandidates">
   <property name="text">
    <string>Candidates</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFace_Recognition">
   <property name="text">
    <string>Face Recognition</string>
   </property>
  </action>
  <action name="actionRestricted_Area">
   <property name="text">
    <string>Restricted Area</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
